pkg/proxy/ipvs/proxier.go

```diff
@@ -131,6 +133,7 @@ type Proxier struct {
 	iptablesData *bytes.Buffer
 	natChains    *bytes.Buffer
 	natRules     *bytes.Buffer
+	syncRunner      *async.BoundedFrequencyRunner // governs calls to syncProxyRules
```

Comment:
nit: please go fmt this file

Reply:
Thanks, I added a new commit to update UT and fixed format problem.